### DISCUSSION TITLE
Copying RTF/HTML to clipboard

### DISCUSSION THREAD

**Discussion by domluther** [dawnlabs/carbon#1552] [Ideas]
Is there anyway to make it so you can export it as some equivalent of RTF so I can paste the code in with the colour coding? At the moment I use SublimeHighlight in Sublime Text to do this as it has a 'copy to clipboard as RTF'. If that were possible on Carbon it would be great.

**Comment by mfix22** [comment]
Hi @domluther, we have gotten this request in the past, and after looking into it, we decided it wouldn't work well within Carbon's system. That said, maybe others have ideas for how to include this feature.
